monitor/api/alerts.py

```diff
@@ -40,21 +42,53 @@ def _query_alert_type(self, run_operation_args: dict, alert_factory_func: Callab
         if raw_alerts:
             alert_dicts = json.loads(raw_alerts[0])
             for alert_dict in alert_dicts:
+                notmalaized_alert = self._normalize_alert(alert=alert_dict)
                 try:
                     alerts.append(alert_factory_func(
                         elementary_database_and_schema=self.elementary_database_and_schema,
-                        **alert_dict
+                        **notmalaized_alert
                     ))
                 except Exception:
                     malformed_alerts.append(MalformedAlert(
-                        alert_dict['id'],
-                        self.elementary_database_and_schema,
-                        alert_dict,
+                        id=notmalaized_alert['id'],
+                        elementary_database_and_schema=self.elementary_database_and_schema,
+                        data=notmalaized_alert,
+                        slack_channel=notmalaized_alert.get('slack_channel')
                     ))
         if malformed_alerts:
             logger.error('Failed to parse some alerts.')
             self.success = False
         return AlertsQueryResult(alerts, malformed_alerts)
+    
+    @classmethod
+    def _normalize_alert(cls, alert: dict) -> dict:
+        try:
+            normalized_alert = copy.deepcopy(alert)
+            meta = try_load_json(normalized_alert.get('meta'))
+            meta = meta if meta else {}
+            model_meta = try_load_json(normalized_alert.get('model_meta'))
+            model_meta = model_meta if model_meta else {}
+
+            subscribers = []
+            direct_subscribers = meta.get('subscribers', [])
+            model_subscribers = model_meta.get('subscribers', [])
+            if isinstance(direct_subscribers, list):
+                subscribers.extend(direct_subscribers)
+            else:
+                subscribers.append(direct_subscribers)
+            if isinstance(model_subscribers, list):
+                subscribers.extend(model_subscribers)
+            else:
+                subscribers.append(model_subscribers)
+
+            slack_channel = model_meta.get('channel')
+
+            normalized_alert['subscribers'] = subscribers
+            normalized_alert['slack_channel'] = slack_channel
+            return normalized_alert
+        except Exception:
+            logger.error(f"Failed to normalaize the alert '{alert.get('id')}'. Might affect the rest of the flow of the alert")
```

Comment:
Maybe say - "Failed to extract alert subscribers and alert custom slack channel {alert.get('id')}. Ignoring it for now and main slack channel will be used"